geopandas/explore.py

```diff
@@ -276,6 +277,22 @@ def my_colormap(value):  # scalar value defined in 'column'
             "'conda install -c conda-forge folium matplotlib mapclassify' "
             "or 'pip install folium matplotlib mapclassify'."
         )
+    # prepare for change in matplotlib
```

Comment:
Can we have this PR focusing on array-like parameters? And leave this for another one.

Reply:
xref #2596